src/current/v25.3/query-behavior-troubleshooting.md

```diff
@@ -109,6 +109,41 @@ docker run -d --name jaeger \
 -p 6831:6831/udp -p 16686:16686 jaegertracing/all-in-one:latest
 ~~~
 
+### Log traces for transactions
+
+CockroachDB allows you to trace [transactions]({% link {{ page.version.version }}/transactions.md %}) to help troubleshoot performance issues. [Tracing]({% link {{ page.version.version }}/show-trace.md %}#trace-description) is controlled through two cluster settings that govern when a transaction trace is captured and emitted.
+
+#### Trace sampling and emission
+
+To enable tracing for a subset of transactions and emit relevant traces to the [`SQL_EXEC` logging channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels), configure the following cluster settings:
```

Comment:
an even more specific link for the `SQL_EXEC` logging channel is https://cockroachlabs.com/docs/v25.3/logging#sql_exec